spec/requests/chatops/slack_slash_command_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.describe "ApplicationChatopsController authentication", type: :request do
```

Comment:
**suggestion**: based on your explanation of auth differences, should this describe be different since this is testing the `SlashCommandController` which happens to have auth?

Reply:
Based on my explanation of copy/paste I should be more careful about editing my test names. 😜
I'll fix.